### PR TITLE
MPDX-7930 - Clean Up Styles on Fix Mailing Addresses

### PR DESCRIPTION
**Pull request by dr-bizz**
## Description
- This pull request focuses on cleaning up styles on the Fix Mailing Addresses page and components to enhance visual consistency and code maintainability.

### Changes
- Added an `LockIcon` styled component.
- Fixed styling for individual contact elements.
- Updated the overall page styling to improve layout and appearance.

## Checklist:

- [ ] I have given my PR a title with the format "MPDX-(JIRA#) (summary sentence max 80 chars)"
- [ ] I have applied the appropriate labels. (_Add the label "On Staging" to get the branch automatically merged into staging._)
- [ ] I have requested a review from another person on the project

### CODE REVIEW

**Comment by dr-bizz** [comment]
@caleballdrin The codecov will come later. This is just styling.

**Comment by caleballdrin** [comment]
@dr-bizz is the amplify preview working for you? It's not working for me.

**Review by caleballdrin — CHANGES_REQUESTED** [review]
Could you make the  Add + icon be aligned with the Lock icons and have the same circle hover effect?

**Comment by dr-bizz** [comment]
@caleballdrin it's due to it building on the staging API. Can you test locally?

File: src/components/Tool/FixMailingAddresses/FixMailingAddresses.tsx [hunk 1]
```diff
@@ -143,98 +148,108 @@ const FixSendNewsletter: React.FC<Props> = ({ accountListId }: Props) => {
...
+                    </Typography>
+                    <Typography>
+                      {t(
+                        'Choose below which mailing address will be set as primary. Primary mailing addresses will be used for Newsletter exports.',
+                      )}
+                    </Typography>
+                    <Box className={classes.defaultBox}>
+                      <Typography>{t('Default Primary Source:')}</Typography>
+
+                      <Select
```
[the remark this answers is not in the document]
Comment (dr-bizz) [reply src/components/Tool/FixMailingAddresses/FixMailingAddresses.tsx:196]: Add icon is going away is a later PR. so I won't fix it.

**Review by caleballdrin — CHANGES_REQUESTED** [review]

File: src/components/Tool/FixMailingAddresses/FixMailingAddresses.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (dr-bizz) [reply src/components/Tool/FixMailingAddresses/FixMailingAddresses.tsx:196]: I've correct the select box change. I got confused and replied to your PR review message here instead.

File: src/components/Tool/FixMailingAddresses/Contact.tsx [hunk 2]
```diff
@@ -121,179 +110,144 @@ const Contact: React.FC<Props> = ({
...
-                    >
-                      <Box
-                        onClick={() => openFunction(newAddress)}
-                        className={classes.address}
-                      />
-                      <Icon path={mdiPlus} size={1} />
+                    </Typography>
+                  </Box>
+                </Grid>
+                <Grid item xs={12} md={6} className={classes.paddingB2}>
```
[the remark this answers is not in the document]
Comment (dr-bizz) [reply src/components/Tool/FixMailingAddresses/Contact.tsx]: Done.

File: src/components/Tool/FixMailingAddresses/Contact.tsx [hunk 3]
```diff
@@ -121,179 +110,144 @@ const Contact: React.FC<Props> = ({
...
-                    <Box
-                      display="flex"
-                      justifyContent="flex-start"
-                      className={clsx(
-                        classes.responsiveBorder,
-                        classes.paddingX,
-                        classes.hoverHighlight,
+                    <Typography>
+                      {address.primaryMailingAddress ? (
+                        <StarIcon className={classes.hoverHighlight} />
```
[the remark this answers is not in the document]
Comment (dr-bizz) [reply src/components/Tool/FixMailingAddresses/Contact.tsx]: Good find

**Comment by caleballdrin** [comment]
Could you make the MPDX Source label aligned with the other sources? And also align the sources with the address on small screen sizes.

**Review by caleballdrin — APPROVED** [review]
Looks great!

**Pull request merged**